swmmio/tests/data/__init__.py

```diff
@@ -31,6 +31,7 @@
 MODEL_EX_1_PARALLEL_LOOP = os.path.join(DATA_PATH, 'Example1_parallel_loop.inp')
 MODEL_INFILTRAION_PARSE_FAILURE = os.path.join(DATA_PATH, 'model-with-infiltration-parse-failure.inp')
 MODEL_EXTCNTRLMODEL = os.path.join(DATA_PATH, 'SWMMExtCntrlModel.inp')
+MODEL_POLYGONS = os.path.join(DATA_PATH, 'example_polygons.inp')
```

Comment:
I don't think we need to add this new inp file to test that this bug fix works properly. Any existing test inp file in the repo with a Polygons section will do.